doc/index.rst

```diff
@@ -1,5 +1,49 @@
-Welcome to Dune's Documentation!
-================================
+Dune Documentation
+==================
+
+Dune is a build system for OCaml projects. Using it, you can build executables,
+libraries, run tests, and much more.
+
+.. grid:: 1 1 1 2
+   :gutter: 2
+
+   .. grid-item-card:: :doc:`getting-started/index`
+
+      **Start here**: a presentation of Dune and how to use it
+
+   .. grid-item-card:: :doc:`howto/index`
+
+      **Step-by-step guides** to help you succeed with Dune
+
+   .. grid-item-card:: :doc:`reference/index`
+
+      **Reference information** about stanzas, actions, DSLs, and commands
+
+   .. grid-item-card:: :doc:`explanation/index`
+
+      **Explanation** of how certain aspects of Dune work
```

Comment:
"Explanation of Dune concepts" maybe? I'm thinking maybe the whole category should rather be called "concepts"?